cmd/erpc/main.go

```diff
@@ -81,7 +84,54 @@ func main() {
 		Name:  "validate",
 		Usage: "Validate the eRPC configuration",
 		Action: baseCliAction(logger, func(ctx context.Context, cfg *common.Config) error {
-			return erpc.AnalyseConfig(cfg, logger)
+			if err := erpc.AnalyseConfig(cfg, logger); err != nil {
+				return err
+			}
+
+			// Ensure telemetry histograms are initialized for validation path
+			var bucketStr string
+			if cfg.Metrics != nil && cfg.Metrics.HistogramBuckets != "" {
+				bucketStr = cfg.Metrics.HistogramBuckets
+			}
+			_ = telemetry.SetHistogramBuckets(bucketStr)
```

Comment:
Shouldn't we return this error instead of silently ignoring? This is basically validating if buckets are correct syntax?